src/ApprenticeCommitments/SFA.DAS.ApprenticeCommitments.MockApis/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I guess this is wrongly included in this PR?

Reply:
Sorry missed that. thx